pennylane_lightning/core/lightning_base.py

```diff
@@ -453,8 +453,8 @@ def execute_and_compute_vjp(
         )
         return tuple(zip(*results))
 
-    # pylint: disable=import-outside-toplevel
-    def eval_jaxpr(self, jaxpr, consts, *args):
+    # pylint: disable=import-outside-toplevel, unused-argument
+    def eval_jaxpr(self, jaxpr, consts, *args, execution_config: Optional[ExecutionConfig] = None):
```

Comment:
Could you please add the new input variable to the docstring?
Also, if possible, could you please type-hint the input variables and function?

Reply:
added.